### PR TITLE
Fix the nightly installation errors

### PR DESCRIPTION
**Pull request by pixiake**
Signed-off-by: pixiake <guofeng@yunify.com>

### CODE REVIEW

File: pkg/kubesphere/manifests.go [hunk 1]
```diff
@@ -416,11 +416,11 @@ spec:
 )
 
 func GenerateKubeSphereYaml(repo, version string) (string, error) {
-	if os.Getenv("KKZONE") == "cn" {
+	if strings.Contains(version, "latest") && os.Getenv("KKZONE") == "cn" {
```
Comment (LinuxSuRen) [inline pkg/kubesphere/manifests.go:419]: How about these versions like v3.0.0, v3.1.0? Actually, I would suggest we sync all images into this registry which include the nightly build image and kubespheredev

Comment (pixiake) [reply pkg/kubesphere/manifests.go:419]: v3.0.0 does not support changing the namespace of images yet. Currently only the latest images are synced, this policy will be adjusted after other images are synchronized.

Comment (LinuxSuRen) [reply pkg/kubesphere/manifests.go:419]: Got you.

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**